src/Zio.Tests/TestSearchPattern.cs

```diff
@@ -107,7 +107,7 @@ public void TestExtensions()
             var newPath = path.ChangeExtension(".zip");
             Assert.Equal("/a/b/c/d.zip", newPath.FullName);
             Assert.Equal(new UPath("a/b/c/d.txt"), path.ToRelative());
-            Assert.Equal(path, path.AssertAbsolute());
+            path.AssertNotNull();
```

Comment:
I left the assert in, as it has to pass, but verifycation is not possible anymore due to the eliminated return

Reply:
reverted